### PR TITLE
Adds the 'Antiseptic Saliva' trait, now with point cost

### PR DESCRIPTION
**Pull request by Mewchild**
Remakes of #4705, but with the trait as a positive, with a cost of 1.

> What it does
Grants access to the 'Lick Wounds' verb, which will target an adjacent creature and bandage/salve any wounds they have on the targeted limb, at the cost of nutrition.
Okay, but why?
I only recently dove into the meaning behind the phrase 'lick your wounds' and finding out that some animals have spit with antiseptic properties really interested me. In addition, I thought that heal bellies could use a non-vore alternative for those without an interest in it.
This should be a positive trait
This is balanced against heal bellies, which are better (soon to be much better) and free for anyone to make and use. It didn't make sense to give this a cost considering there was a free and more effective alternative. Since it can only be used on others, I'd expect this to see sporadic use by people with character motivations that align with it.

### CODE REVIEW

**Comment by Very-Soft** [comment]
While everyone gets healbelly, healbelly doesn't presently do what this does. Making it cost a point to specialize in healing people properly makes sense. 

I don't really care one way or the other, but bandaging and disinfecting wounds with no supplies is generally more actually practically useful in more situations than healbelly, and a lick is waaaaay less likely to be a prefbreak for people.

Of course it might also be neat to make healbelly a trait too but that's just me pondering.

**Comment by SpadesNeil** [comment]
I don't want to make healbelly a trait, but the simple addition of this makes people able to be mobile doctors short of healing broken bones or damaged organs.

It would make healing bellies part of the meta again, which I kind of miss from old school Vorestation.

**Comment by Haery70** [comment]
I wanted to say that I didn't mean to start anything in the previous pr and I apologize. I waited until everything was cooled down and understood the motive behind bringing the pr to here. I dunno, something in the previous pr rattled my mind or something that got me in an antsy mood. By judging from the code before, the whole 'free positive trait' thing was the major thing that irked me about it, which could encourage the powergaming trait issue that we've had. But, since the trait has been altered to be more reasonable, I'm more than willing to accept this.

**Comment by Very-Soft** [comment]
I do think that this trait will make heal belly much more viable in medical doings, so I think it's good~

**Comment by SpadesNeil** [comment]
Now we still have a feature freeze so I won't merge this, but this will probably get reviewed and merged when we're done with said freeze.

**Comment by Wickedtemp** [comment]
Not going to lie, I'm going to use this trait as much as I can, even if it's bumped up to 2 trait points. Not sure how I'll minmax for the spare points, but I'll find a way.

Medical mlems are worth it.

**Comment by CameronWoof** [comment]
You will want to turn this into a proc instead of a verb (just change the path from /verb/ to /proc/ in both files). 

Turns out, verbs are automatically added to every player, trait or no, and you probably didn't want that!

**Comment by Novacat** [comment]
I still think this should also be available on some vanilla species, to counter the argument about stuff like this making custom species blatantly better.

File: code/modules/mob/living/carbon/human/species/station/traits_vr/positive.dm [hunk 1]
```diff
@@ -109,3 +109,12 @@
...
 	cost = 1
 	var_changes = list("flags" = NO_MINOR_CUT) //Checked the flag is only used by shard stepping.
+
+/datum/trait/antiseptic_saliva
+	name = "Antiseptic Saliva"
+	desc = "Your saliva has especially strong antiseptic properties that can be used to heal small wounds."
+	cost = 1
+
+/datum/trait/antiseptic_saliva/apply(var/datum/species/S,var/mob/living/carbon/human/H)
+	..(S,H)
```
Comment (kevinz000) [inline code/modules/mob/living/carbon/human/species/station/traits_vr/positive.dm]: just . = ..(), args are copied automagically.

File: code/modules/mob/living/carbon/lick_wounds.dm [hunk 2]
```diff
@@ -0,0 +1,78 @@
+/mob/living/carbon/human/verb/lick_wounds(var/mob/living/carbon/M in living_mobs(1))
+	set name = "Lick Wounds"
+	set category = "Abilities"
+	set desc = "Disinfect and heal small wounds with your saliva."
+
+	if(src.nutrition < 50)
```
Comment (kevinz000) [inline code/modules/mob/living/carbon/lick_wounds.dm]: src. isn't needed

File: code/modules/mob/living/carbon/lick_wounds.dm [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+			istype(src, /mob/living/silicon)) )
+		src << "<span class='warning'>If you even have a tongue, it doesn't work that way.</span>"
+		return
+
+	if (istype(M, /mob/living/carbon/human))
+		var/mob/living/carbon/human/H = M
+		var/obj/item/organ/external/affecting = H.get_organ(src.zone_sel.selecting)
+
+		if(!affecting)
+			src << "<span class='warning'>No body part there to work on!</span>"
```
Comment (kevinz000) [inline code/modules/mob/living/carbon/lick_wounds.dm]: to_chat

File: code/modules/mob/living/carbon/lick_wounds.dm [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+			return
+
+		if(affecting.organ_tag == BP_HEAD)
+			if(H.head && istype(H.head,/obj/item/clothing/head/helmet/space))
+				src << "<span class='warning'>You can't seem to lick through [H.head]!</span>"
```
Comment (kevinz000) [inline code/modules/mob/living/carbon/lick_wounds.dm]: to_chat

File: code/modules/mob/living/carbon/lick_wounds.dm [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+				return
+
+		else
+			if(H.wear_suit && istype(H.wear_suit,/obj/item/clothing/suit/space))
+				src << "<span class='warning'>You can't lick your way through [H.wear_suit]!</span>"
```
Comment (kevinz000) [inline code/modules/mob/living/carbon/lick_wounds.dm]: to_chat

File: code/modules/mob/living/carbon/lick_wounds.dm [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+				return
+
+		if(affecting.robotic == ORGAN_ROBOT)
+			src << "<span class='warning'>You don't think your spit will help a robotic limb.</span>"
```
Comment (kevinz000) [inline code/modules/mob/living/carbon/lick_wounds.dm]: to_chat

File: code/modules/mob/living/carbon/lick_wounds.dm [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+			return
+
+		if(affecting.robotic >= ORGAN_LIFELIKE)
+			src << "<span class='warning'>You lick [M]'s [affecting.name], but it seems to have no effect...</span>"
```
Comment (kevinz000) [inline code/modules/mob/living/carbon/lick_wounds.dm]: to_chat

File: code/modules/mob/living/carbon/lick_wounds.dm [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+		if(affecting.open)
+			to_chat(src, "<span class='notice'>The [affecting.name] is cut open, you don't think your spit will help them!</span>")
+			return
+
+		if(affecting.is_bandaged() && affecting.is_salved())
+			to_chat(src, "<span class='warning'>The wounds on [M]'s [affecting.name] have already been treated.</span>")
+			return
+
+		else
+			src.visible_message("<span class='notice'>\The [src] starts licking the wounds on [M]'s [affecting.name] clean.</span>", \
```
Comment (kevinz000) [inline code/modules/mob/living/carbon/lick_wounds.dm]: src. isn't needed

File: code/modules/mob/living/carbon/lick_wounds.dm [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+				if(!do_mob(src, M, W.damage/5))
+					to_chat(src, "<span class='notice'>You must stand still to clean wounds.</span>")
+					break
+
+				if(affecting.is_bandaged() && affecting.is_salved()) // We do a second check after the delay, in case it was bandaged after the first check.
+					to_chat(src, "<span class='warning'>The wounds on [M]'s [affecting.name] have already been treated.</span>")
+					return 
+
+				else
+					src.visible_message("<span class='notice'>\The [src] [pick("slathers \a [W.desc] on [M]'s [affecting.name] with their spit.",
```
Comment (kevinz000) [inline code/modules/mob/living/carbon/lick_wounds.dm]: src. isn't needed

File: code/modules/mob/living/carbon/lick_wounds.dm [hunk 10]
```diff
@@ -0,0 +1,78 @@
...
+					return 
+
+				else
+					src.visible_message("<span class='notice'>\The [src] [pick("slathers \a [W.desc] on [M]'s [affecting.name] with their spit.", 
+																			   "drags their tongue across \a [W.desc] on [M]'s [affecting.name].",
+																			   "drips saliva onto \a [W.desc] on [M]'s [affecting.name].",
+																			   "uses their tongue to disinfect \a [W.desc] on [M]'s [affecting.name].",
+																			   "licks \a [W.desc] on [M]'s [affecting.name], cleaning it.")]</span>", \
+					                        	"<span class='notice'>You treat \a [W.desc] on [M]'s [affecting.name] with your antiseptic saliva.</span>" )
+					src.nutrition -= 20
```
Comment (kevinz000) [inline code/modules/mob/living/carbon/lick_wounds.dm]: src. isn't needed

**Comment by Mewchild** [comment]
I don't know what I'm doing
`code\modules\mob\living\carbon\lick_wounds.dm:20:error: to_chat: undefined var
code\modules\mob\living\carbon\lick_wounds.dm:25:error: to_chat: undefined var
code\modules\mob\living\carbon\lick_wounds.dm:30:error: to_chat: undefined var
code\modules\mob\living\carbon\lick_wounds.dm:34:error: to_chat: undefined var
code\modules\mob\living\carbon\lick_wounds.dm:38:error: to_chat: undefined var`

**Comment by Sebbe9123** [comment]
> I don't know what I'm doing
> `code\modules\mob\living\carbon\lick_wounds.dm:20:error: to_chat: undefined var code\modules\mob\living\carbon\lick_wounds.dm:25:error: to_chat: undefined var code\modules\mob\living\carbon\lick_wounds.dm:30:error: to_chat: undefined var code\modules\mob\living\carbon\lick_wounds.dm:34:error: to_chat: undefined var code\modules\mob\living\carbon\lick_wounds.dm:38:error: to_chat: undefined var`

to_chat(src, "message")
(M might work instead of src, both compiles)

**Comment by Mewchild** [comment]
Oh no I fucked everything.

**Comment by Mewchild** [comment]
Lemme just remake this, fresh.

**Pull request closed without merging**